app/app.go

```diff
@@ -56,6 +58,20 @@ type FrameEvent struct {
 	Source input.Source
 }
 
+// URLEvent is generated when the program is invoked with a URL. This event is delivered
+// through Event iterator and not through a Window.Event.
+//
+// In order to receive the program must register a scheme. The scheme can be registered
+// using gogio, with the `-schemes` flag.
+//
+// Due to differences between OSes, URLEvent might not be generated when the URL is
+// larger than 2048 characters. Domain names using non-ASCII characters have
+// undefined behavior, it can be either encoded as Punycode, QueryEscape,
```

Comment:
Saying "undefined behaviour" and then listing possible encodings is not great for the user. Also, which platform(s) encodes non-ASCII domains as "QueryEscape"? Can we decode those to UTF-8, and then say

"Domain names using non-ASCII characters may be encoded in UTF-8 or punycode"

?

In other words, I'd like to expose only the minimal possible encodings to the user. *someone* has to bear the decoding burden and it might as well be Gio.